api/middleware/validation.js

```diff
@@ -1,5 +1,6 @@
 const {
   incidentQuery: incidentQuerySchema,
+  incidentObjectSchema,
 } = require('../util/validationSchemas');
 
 exports.validateAndSanitizeIncidentQueries = async (req, _res, next) => {
```

Comment:
Reasoning behind _res?

Reply:
`_` to signal to coders and linter that `res` will not be used in the function but is necessary as a parameter because the following parameter `next` will be used in the function.

Reply:
Good idea - this will definitely help future collaborators.